Website/src/rendering/src/pages/shop/checkout/cart.tsx

```diff
@@ -11,19 +16,29 @@ interface CartProps {
 }
 
 const Cart = (props: CartProps): JSX.Element => {
+  const { order } = useOcCurrentOrder();
+  const breadCrumbDefinitions: Breadcrumb[] = [];
   return (
     <ShopLayout>
       {console.log({ props })}
       <Head>
         <title>PLAY! SHOP - Cart</title>
       </Head>
 
-      <p>Cart Page</p>
+      <Breadcrumb
+        rendering={{ componentName: '' }}
+        params={{}}
+        fields={{ items: breadCrumbDefinitions }}
+      />
+      <h1>Your Shopping Cart</h1>
+      <LineItemList editable={true} />
       <div>
         <Link href="/shop/checkout/quick-checkout">
           <a>Go to Checkout</a>
         </Link>
       </div>
+      {order && <GiftCheckboxOrder order={order} />}
```

Comment:
This page will be converted to a Sitecore item with a list of components in placeholders in the future. We will not be able to add conditions like this at the page level to only render some components if there is an order variable defined.

Could the logic to display or not the component markup be moved into each individual component instead?